3.0/Dockerfile.fedora

```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/f34/s2i-base
+FROM registry.fedoraproject.org/f35/s2i-base
```

Comment:
We have a new `s2i-base` container which is updated regularly each week.
The reference is `quay.io/fedora/s2i-base:35` https://quay.io/repository/fedora/s2i-base?tab=tags

Can you please update this PR with this new `s2i-base` container?